core/src/migrations/000099-addIncrementalToSchedules.ts

```diff
@@ -0,0 +1,36 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.addColumn("schedules", "incremental", {
+      type: DataTypes.BOOLEAN,
+      allowNull: true,
+      defaultValue: true,
+    });
+
+    const [nonIncrementalSources]: [Record<string, any>[], unknown] =
+      await queryInterface.sequelize.query(
+        `SELECT * FROM "sources" WHERE type IN ('csv-import-table', 'google-sheet-import')`
+      );
+
+    if (nonIncrementalSources.length > 0) {
+      await queryInterface.sequelize.query(
+        `UPDATE schedules SET incremental=false WHERE id IN (${nonIncrementalSources
+          .map((s) => s.id)
+          .join(", ")})`
+      );
+    }
```

Comment:
Any non-incremental sources I missed?

Reply:
calculated property?
sql queries ones for sure aren't incremental.